webservices/resources/reports.py

```diff
@@ -105,11 +134,18 @@ def build_query(self, committee_id=None, committee_type=None, **kwargs):
         if kwargs.get('is_amended') is not None:
             query = query.filter(reports_class.is_amended == kwargs['is_amended'])
 
+        query = filters.filter_range(query, kwargs, self.filter_range_fields)
         return query, reports_class, reports_schema
 
     def _resolve_committee_type(self, committee_id=None, committee_type=None, **kwargs):
-        if committee_id is not None:
-            query = models.CommitteeHistory.query.filter_by(committee_id=committee_id)
+        #if kwargs.get('committee_id') is not None and len(kwargs.get('committee_id')) > 0:
+        if committee_id is not None or kwargs.get('committee_ids'):
+            comm_id = (
+                committee_id
+                if committee_id is not None
+                else kwargs.get('committee_ids')[0]
```

Comment:
So, the structure of the front-end matches the structure of the API. There's no top-level /reports/ page, there's just: `/reports/presidential` `/reports/house-senate/` `/reports/pac-party/` and `/reports/ie-only`

So yeah, if someone requested a `committee_id` of a committee that didn't match the current page, nothing would show up. And if they requested a `candidate` with authorized committees not of the current page, nothing would show up.

Whether or not we accept multiple IDs is kind of beside the point, since even accepting one ID of a non-matching type, won't return anything.

On the one hand, this is a pretty advanced feature and I think if we don't resolve this now, it's not a _huge_ deal. I think we can expect advanced users to recover.

But from a UX perspective, I think the best solution would be to just update the `names` endpoint to scope to candidates or committees of certain types. This is the endpoint that the typeahead hits, so if there was something like `/names/candidates/presidential/` that only returned presidential candidates, or `/names/committees/house-senate/` that only returned H & S type commmittees, that might work well.